src/main/java/com/tilmenk/warehouse/team/Team.java

```diff
@@ -0,0 +1,113 @@
+package com.tilmenk.warehouse.team;
+
+import com.tilmenk.warehouse.pokemon.Pokemon;
+
+import javax.persistence.*;
+
+@Entity
+@Table
+public class Team {
+
+    @Id
+    @SequenceGenerator(name = "student_sequence", sequenceName =
+            "student_sequence", allocationSize = 1)
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator =
+            "student_sequence")
+    private Long id;
+
+    @ManyToOne
+    private Pokemon pokemon_0;
```

Comment:
I think this could be another relationship, where you implement some limits on one end (e.g. only max. 6 pokemon in a team)

Reply:
Thanks for your suggestions! I'll ask you about this later in the Uebung :)